public/components/SponsorshipEdit/SponsorEdit.react.js

```diff
@@ -37,6 +38,9 @@ export default class SponsorEdit extends React.Component {
       return false;
     }
 
+    const logoWidth = this.props.paidContentTagType === PAID_HOSTEDCONTENT_TYPE.value ? false : 140;
+    const logoHeight = this.props.paidContentTagType === PAID_HOSTEDCONTENT_TYPE.value ? false : 90;
```

Comment:
It's either this or add more logic into the support method on the server, and pass tag information with the request... I know what you mean though